src/test/java/uk/gov/pay/adminusers/resources/UserResourceGetIT.java

```diff
@@ -18,10 +19,52 @@
 import static uk.gov.pay.adminusers.fixtures.ServiceDbFixture.serviceDbFixture;
 import static uk.gov.pay.adminusers.fixtures.UserDbFixture.userDbFixture;
 
-public class UserResourceGetIT extends IntegrationTest {
+class UserResourceGetIT extends IntegrationTest {
 
     @Test
-    public void shouldReturnUser_whenGetUserWithExternalId() {
+    void should_return_empty_map_when_getting_admin_emails_for_gateway_accounts() {
+        givenSetup()
+                .when()
+                .contentType(JSON)
+                .accept(JSON)
+                .queryParam("gatewayAccountId", "gatewayAccount")
+                .get("/v1/api/users/admin-emails-for-gateway-accounts")
+                .then()
+                .statusCode(200)
+                .body("gatewayAccount", hasSize(0));
+    }
+    
+    @Test
+    void should_return_admin_emails_for_gateway_accounts() {
+        String gatewayAccount1 = valueOf(nextInt());
+        String gatewayAccount2 = valueOf(nextInt());
+        Service service = serviceDbFixture(databaseHelper).withGatewayAccountIds(gatewayAccount1, gatewayAccount2)
+                .insertService();
+        
+        Role adminRole = roleDbFixture(databaseHelper).insertAdmin();
+        var adminUser1 = userDbFixture(databaseHelper).withServiceRole(service.getId(), adminRole.getId()).insertUser();
+        var adminUser2 = userDbFixture(databaseHelper).withServiceRole(service.getId(), adminRole.getId()).insertUser();
+        
+        Role viewOnlyRole = roleDbFixture(databaseHelper).withName("view-only").insertRole();
+        userDbFixture(databaseHelper).withServiceRole(service.getId(), viewOnlyRole.getId()).insertUser();
+
+        givenSetup()
+                .when()
+                .contentType(JSON)
+                .accept(JSON)
+                .queryParam("gatewayAccountId", gatewayAccount1)
+                .queryParam("gatewayAccountId", gatewayAccount2)
```

Comment:
The request will look something like `/v1/api/users/admin-emails-for-gateway-accounts?gatewayAccountId=206979493&gatewayAccountId=429850515`